<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodesManagementWidget</class>
 <widget class="QWidget" name="NodesManagementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>1110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="MultiView" name="widgetIncommingNodes" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="MultiView" name="widgetActiveNodes" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButtonAddLocal">
           <property name="text">
            <string>Add...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEditSummary">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="backgroundVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonShowStats">
         <property name="text">
          <string>Stats</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionAddAgent">
   <property name="icon">
    <iconset resource="../../libres/resources/icons.qrc">
     <normaloff>:/icons/app/window/agent.svg</normaloff>:/icons/app/window/agent.svg</iconset>
   </property>
   <property name="text">
    <string>Agent</string>
   </property>
   <property name="toolTip">
    <string>Instanciate an agent</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAddRemote">
   <property name="icon">
    <iconset resource="../../libres/resources/icons.qrc">
     <normaloff>:/icons/app/window/remote.svg</normaloff>:/icons/app/window/remote.svg</iconset>
   </property>
   <property name="text">
    <string>Remote</string>
   </property>
   <property name="toolTip">
    <string>Instanciate a remote</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MultiView</class>
   <extends>QWidget</extends>
   <header location="global">hub/MultiView.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../libres/resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonAddLocal</sender>
   <signal>clicked()</signal>
   <receiver>NodesManagementWidget</receiver>
   <slot>showAddLocal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddAgent</sender>
   <signal>triggered()</signal>
   <receiver>NodesManagementWidget</receiver>
   <slot>addLocalAgent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddRemote</sender>
   <signal>triggered()</signal>
   <receiver>NodesManagementWidget</receiver>
   <slot>addLocalRemote()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showAddLocal()</slot>
  <slot>addLocalAgent()</slot>
  <slot>addLocalRemote()</slot>
 </slots>
</ui>
